src/index.js

```diff
@@ -1,17 +1,31 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { BrowserRouter as Router } from 'react-router-dom';
+
+import { Provider } from 'react-redux';
+
+import { library } from '@fortawesome/fontawesome-svg-core';
+import {
+  faAngleLeft, faMicrophone, faGear, faCircleArrowRight,
+} from '@fortawesome/free-solid-svg-icons';
+
+import store from './redux/configureStore';
+
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 import './bootstrap';
 
+library.add(faAngleLeft, faMicrophone, faGear, faCircleArrowRight);
+
 const githubBasename = process.env.REACT_APP_BASENAME || '/';
 
 ReactDOM.render(
-  <React.StrictMode basename={githubBasename}>
-    <Router>
-      <App />
-    </Router>
+  <React.StrictMode>
+    <Provider store={store}>
+      <Router basename={githubBasename}>
+        <App />
+      </Router>
+    </Provider>
```

Comment:
The project uses Redux for state management and data retrieved from the API are stored in the Redux store correctly 😉 Fantastic job!